pipeline/inputs/standard-input.md

```diff
@@ -48,11 +67,56 @@ Fluent Bit v1.x.x
 [4] stdin.0: [1475898290, {"key"=>"some value"}]
 ```
 
+### Parser input example <a id="parser-input-example"></a>
+
+To capture inputs in other formats, specify a parser configuration for the
+`stdin` plugin.
+
+For example, if you want to read raw messages line-by-line and forward them you
+could use a `parser.conf` that captures the whole message line:
+
+```
+[PARSER]
+    name        stringify_message
+    format      regex
+    Key_Name    message
+    regex       ^(?<message>.*)
+```
+
+then use that in the `parser` clause of the stdin plugin in the `fluent-bit.conf`:
+
+```
+[INPUT]
+    Name    stdin
+    Tag     stdin
+    Parser  stringify_message
+
+[OUTPUT]
+    Name   stdout
+    Match  *
+```
+
+`fluent-bit` will now read each line and emit a single message for each input
+line:
+
+```
+$ seq 1 5 | /opt/fluent-bit/bin/fluent-bit -c fluent-bit.conf -R parser.conf -q
+[0] stdin: [1681358780.517029169, {"message"=>"1"}]
+[1] stdin: [1681358780.517068334, {"message"=>"2"}]
+[2] stdin: [1681358780.517072116, {"message"=>"3"}]
+[3] stdin: [1681358780.517074758, {"message"=>"4"}]
+[4] stdin: [1681358780.517077392, {"message"=>"5"}]
+$
+```
+
+In most cases you'll want to use a more realistic parser that splits your
```

Comment:
Try not to use shortenings of words so `you'll` should be `you will`. I would also try to avoid any kind of personal pronoun like `you`.

Reply:
Reworded to

> In real-world deployments it is best to use a more realistic parser that splits               
> messages into real fields and adds appropriate tags.